src/leet2git/leetcode_client.py

```diff
@@ -34,13 +34,13 @@ def get_cookies(self) -> Tuple[str, str]:
         url = "https://leetcode.com/profile/"
         client = requests.session()
         try:
-            browsers = (browser_cookie3.chrome(), browser_cookie3.firefox())
+            browsers = (browser_cookie3.chrome, browser_cookie3.firefox)
         except browser_cookie3.BrowserCookieError as e:
             click.secho(e.args, fg="red")
 
         for browser in browsers:
             try:
-                r = client.get(url, cookies=browser)
+                r = client.get(url, cookies=browser())
```

Comment:
I think lazy call does not work in my case.
In my case, it fails to get csrftoken while getting cookies is fine.

Reply:
As it fails to initialize leetcode client, none of subsequent task does not work.